newsroom/auth/utils.py

```diff
@@ -120,17 +126,49 @@ def is_valid_session():
     )
 
 
+def get_user_company(user) -> Optional[Dict]:
```

Comment:
This is already defined in `newsroom.companies.utils`
https://github.com/superdesk/newsroom-core/blob/develop/newsroom/companies/utils.py#L4

Reply:
I've added this due to a circular import it seems, will be doing some more refactoring in a following PR so will handle it there